crates/telio-wg/src/adapter/mod.rs

```diff
@@ -190,6 +197,7 @@ pub(crate) fn start(
     socket_pool: Arc<SocketPool>,
     firewall_process_inbound_callback: FirewallCb,
     firewall_process_outbound_callback: FirewallCb,
+    firewall_icmp_kill_conns_callback: FirewallResetConnsCb,
```

Comment:
As we are not using ICMP only for killing of connections, maybe it would be better to:
```suggestion
    firewall_kill_conns_callback: FirewallResetConnsCb,
```

Reply:
Oh, yeah. It's a leftover

Reply:
Done